packages/fether-electron/src/main/app/methods/setupWinListeners.js

```diff
@@ -13,10 +12,18 @@ const pino = Pino();
 function setupWinListeners (fetherApp) {
   const { onWindowClose, processSaveWinPosition, win } = fetherApp;
 
-  // Open external links in browser
-  win.webContents.on('new-window', (event, url) => {
-    event.preventDefault();
-    electron.shell.openExternal(url);
+  // Insecure TLS Validation - verify the application does not explicitly opt-out of TLS validation
+  // Reference: https://doyensec.com/resources/us-17-Carettoni-Electronegativity-A-Study-Of-Electron-Security-wp.pdf
+  win.webContents.session.setCertificateVerifyProc((request, callback) => {
+    const { hostname } = request;
+
+    if (hostname === 'https://localhost:3000/') {
+      // Success and disables certificate verification
```

Comment:
Same as above: can we leave it enabled at all times?

Reply:
No, I don't think we should. I've gone through the API docs https://electronjs.org/docs/api/session#sessetcertificateverifyprocproc, and pushed a commit with some proposed updates